### ISSUE TITLE
Confusing error message downloading crypto

### ISSUE THREAD

**Issue by deepakjois** [coursera-dl/coursera-dl#460]
```
:~$ coursera-dl -n --aria2  -- crypto
Downloading class: crypto
Starting new HTTPS connection (1): class.coursera.org
Starting new HTTPS connection (1): www.coursera.org
Logged in on coursera.org.
Starting new HTTPS connection (1): class.coursera.org
Found authentication cookies.
Downloading about page from: https://api.coursera.org/api/catalog.v1/courses?fields=largeIcon,photo,previewLink,shortDescription,smallIcon,smallIconHover,universityLogo,universityLogoSt,video,videoId,aboutTheCourse,targetAudience,faq,courseSyllabus,courseFormat,suggestedReadings,instructor,estimatedClassWorkload,aboutTheInstructor,recommendedBackground,subtitleLanguagesCsv&q=search&query=crypto
Starting new HTTPS connection (1): api.coursera.org
Subtitle unavailable in specified language
Downloaded https://class.coursera.org/crypto/lecture/index (766095 bytes)
Found 0 sections and 0 lectures on this page
The cookies file may be invalid, please re-run with the `--clear-cache` option.
```

Looking at the API:
```
$ curl "https://www.coursera.org/api/onDemandCourses.v1?&q=slug&slug=crypto"
…
    "subtitleLanguageCodes": [
        "pt"
      ],
…
```

So I guess the default language is English, and this course either doesn’t have or doesn’t advertise subtitles in English. But when I look at the actual video, it shows English and Ukranian as options as well. So maybe something is wrong with the course data.

**Comment by deepakjois** [comment]
Another course having a similar error – security

```
$ $ curl "https://www.coursera.org/api/onDemandCourses.v1?&q=slug&slug=security"
…
"subtitleLanguageCodes": [],
…
```

**Comment by FiloSottile** [comment]
The crypto error is fixed by #496
